specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -591,15 +693,15 @@ SECRET_VALUE.
 
   @doc("The creation datetime(ISO 8601 literal format).")
   @visibility("read")
-  createdDateTime?: offsetDateTime;
+  createdDateTime?: utcDateTime;
```

Comment:
can this be defined as a seperate model as SystemMetadata and included for both Test, TestRun etc.